cinq_auditor_required_tags/providers.py

```diff
@@ -216,7 +223,7 @@ def delete_s3_bucket(client, resource):
             try:
                 rules = bucket.LifecycleConfiguration().rules
                 for rule in rules:
-                    if rule['ID'] == 'cloudInquisitor':
+                    if rule['ID'] == 'cloudInquisitor2':
```

Comment:
@TheM0ng00se Can this be renamed to be more descriptive? e.g. `cinqExpiredObjectsDeleteMarker`

Reply:
Yeah good point. I wanted to guarantee it was enumerated after the first rule ID but guess us iterating over the rules doesn't matter much since we apply the rules as a 'ruleset' so it finding any of these rule IDs would satisfy we have both rules applied.